lib/ansible/executor/module_common.py

```diff
@@ -1356,7 +1356,7 @@ def get_action_args_with_defaults(action, args, defaults, templar, redirected_na
 
             group_name = default.split('group/')[-1]
 
-            for collection_name in group_collection_map[group_name]:
+            for collection_name in group_collection_map.get(group_name, []):
```

Comment:
Is that behavior consistent with 2.9? At first blush, I'd expect a totally bogus group name to be a fatal error- if it's not, maybe it should be...

Reply:
Yes, it's consistent with 2.9. module_defaults has never been validated.